src/Components/ErrorsNotify/ErrorsNotify.tsx

```diff
@@ -0,0 +1,42 @@
+import { useContext, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { TodoContext } from '../TodoContext/TodoContext';
+
+export const ErrorsNotify = () => {
+  const { errorMessage, setErrorMessage } = useContext(TodoContext);
+  const [showErrorMessage, setShowErrorMessage] = useState(false);
```

Comment:
```suggestion
  const showErrorMessage = !!errorMessage;
```
empty string is [falsy](https://www.freecodecamp.org/news/what-are-falsey-values-in-javascript/) value